ToDoApp/ToDoApp/ToDoListView.swift

```diff
@@ -13,10 +13,11 @@ struct ToDoSection: Identifiable {
     let toDoItems: [TaskModel]
 }
 struct ToDoListView: View {
-    @StateObject var viewModel = ToDoListViewModel()
+    @StateObject var viewModel: ToDoListViewModel = ToDoListViewModel()
+    var addButttonSize: Double = 57.0
```

Comment:
a minimum requirement for defining constants is that they should be defined as let 

there is also a common pattern of defining constants in swift which is the following:
define an enum in a type private extension named Constants, with static stored constants like this:
```
private extension ToDoListView {
    enum Constants {
        static let addButtonSize: CGFloat = 57
    }
}
```
answer on a question why would we use enum you may find here: https://www.swiftbysundell.com/articles/powerful-ways-to-use-swift-enums/

Reply:
This has been fixed.
And I have a question, so whenever we create have a constant value and want to define, we can't do something like let constantValue: Double = 57.0?

Reply:
We can, as I said, this is a minimum requirement. But having an enum wrapper around defined constants is another common pattern and many beginners are not aware of it, since this a learning project, the main purpose is to learn something new, right? 🙂